src/main/webapp/i18n/en/lti.json

```
@@ -87,6 +87,7 @@
                 "linkedSuccessfully": "Linked exercises successfully",
                 "linkedFailed": "Error during deep linking",
                 "link": "Link",
+                "selectToLink": "Select exercises to link",
```

Since it's an error message for the user:

```suggestion
                "selectToLink": "You have to select exercises to link.",
```

---

It's not only an error but also the button text, thus I figured a shorter error message is fine. Moreover, the alert should never be shown, as the button should not be clickable without a selected exercise